website/blog/2024-10-29-kubeflow-setup/index.mdx

```diff
@@ -0,0 +1,454 @@
+---
+slug: kubeflow-setup-guide
+title: Getting Started with Cloud-native AI using Kubeflow
+description: |
+  In this article, I would like to introduce to you Kubeflow: A complete and cloud-native platform that simplifies AI operations.
+  Join me in setting up Kubeflow on GKE for your organization and get started with cloud-native AI today.
+authors:
+  - kosmoz
+tags: [guide, ai, mlops, kubernetes, eks, gcp, azure]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In this article, I would like to introduce to you Kubeflow: A complete and cloud-native platform that simplifies AI operations.
+Join me in setting up Kubeflow on GKE for your organization and get started with cloud-native AI **today**.
+
+{/* truncate */}
+
+## Prerequisites
+
+- Up-to-date version of `git`, `kubectl` and `kustomize` installed
```

Comment:
Might be interesting to include a backlink somewhere to link to the Kustomize guide: https://glasskube.dev/blog/patching-with-kustomize/

Reply:
That's a great idea. I think I'll add one closer to the end where the article suggests that people should take this as a basis to build their own kubeflow from